.github/workflows/ci.yml

```diff
@@ -759,10 +766,10 @@ jobs:
         else
             if [[ "${{ matrix.clang-runtime }}" == "17" || "${{ matrix.clang-runtime }}" == "18" ]]; then
```

Comment:
We do not seem to have an llvm 18 build. Can you add one? I think https://github.com/llvm/llvm-project/issues/90506 resolves the valgrind issues on 18.

Reply:
We need to exclude 18 from the `|| true` rule.